src/otaclient/configs/_cfg_configurable.py

```diff
@@ -62,8 +60,7 @@ class _OTAClientSettings(BaseModel):
     #
     # ------ create standby settings ------ #
     #
-    CREATE_STANDBY_METHOD: CREATE_STANDBY_METHOD_LTIERAL = "REBUILD"
```

Comment:
Don't expose the option of choosing which create_standby method to use now.
Detect automatically by `create_standby.utils.can_use_in_place_mode` helper, and prefer `inplace` mode if possible.